packages/ubuntu_desktop_installer/lib/routes.dart

```diff
@@ -6,4 +6,5 @@ abstract class Routes {
   static const tryUbuntu = '/tryubuntu';
   static const allocateDiskSpace = '/allocatediskspace';
   static const updatesOtherSoftware = '/updateothersoftware';
+  static const choseYourLook = 'choseyourlook';
```

Comment:
nit-pick: typo

Reply:
Oh man this wird really can't get into my brain xD sorry! And thanks, will correct!